extensions/2.0/Vendor/EXT_lights_image_based/schema/light.schema.json

```diff
@@ -51,15 +51,15 @@
             "minItems": 1
         },
         "specularImageSize": {
-            "type": "number",
-            "description": "The maximum dimension (in pixels) of the first specular mip. This is needed to determine, pre-load, the total number of mips needed.",
-            "minimum": 0
+            "type": "integer",
+            "description": "The dimension (in pixels) of the first specular mip. This is needed to determine, pre-load, the total number of mips needed.",
+            "minimum": 0,
```

Comment:
Couldn't this just be `1` instead of `0` with `exclusiveMinimum`?

Reply:
Done